ui/packages/catalog-ui-search/src/main/webapp/component/search-form-editor/search-form-editor.less

```diff
@@ -0,0 +1,16 @@
+@{customElementNamespace}search-form-editor {
+  > .search-form-editor-title {
```

Comment:
this looks like a duplicate of the inline styling you use in ```search-form-editor.hbs```

Reply:
Andrew brought that up. Initially, I put it in the hbs file, but he told me to pull it out. I put it in the .less file and forgot to remove it from the hbs file.

Reply:
fixed